osu.Game/Rulesets/Mods/ModFlashlight.cs

```diff
@@ -127,13 +133,8 @@ protected override void LoadComplete()
 
                 using (BeginAbsoluteSequence(0))
                 {
-                    foreach (var breakPeriod in Breaks)
+                    foreach (var breakPeriod in Breaks.Where(breakPeriod => breakPeriod.HasEffect && breakPeriod.Duration >= FLASHLIGHT_FADE_DURATION * 2))
```

Comment:
Prefer how it was previously, don't like combining all conditions in one big `Where` operation.